system-addon/content-src/components/Search/_Search.scss

```diff
@@ -184,7 +184,6 @@
   input {
     @include search-input;
     border-top-left-radius: $search-border-radius;
-    border-bottom-left-radius: $search-border-radius;
     padding-inline-start: 35px;
 
     &:focus + button {
```

Comment:
Ah doh. I thought I didn't need to change `button` to `.search-button` here as it's effectively scoped to just the actual search button, but CSS specificity calculations with the element caused styling to get messed up relative to the lower specificity of the class selector.